aws/resource_aws_lb_target_group.go

```diff
@@ -46,27 +47,27 @@ func resourceAwsLbTargetGroup() *schema.Resource {
 				Computed:      true,
 				ForceNew:      true,
 				ConflictsWith: []string{"name_prefix"},
-				ValidateFunc:  validateAwsLbTargetGroupName,
+				ValidateFunc:  validateMaxLength(32),
 			},
 			"name_prefix": {
 				Type:         schema.TypeString,
 				Optional:     true,
 				ForceNew:     true,
-				ValidateFunc: validateAwsLbTargetGroupNamePrefix,
+				ValidateFunc: validateMaxLength(32 - 26),
```

Comment:
add a comment about why this calculation

Reply:
Can we leave it without the comment for now? When all PRs get merged, I can add comment to such ValidateFuncs all together.

Reply:
Hey @loivis 

I'm also in favor in embedding this kind of comments right away not to forget about it afterwards.

Reply:
I understand and agree as well. Comments are needed across multiple branches for exactly the same thing. In this case, adding comments after all merges sounds better to me. Furthermore, @paultyng has created a pr adding const in `validation` package. Eventually it would end up like `validateMaxLength(32 - validation.UniqueIDSuffixLength)`. I won't leave it behind.

Reply:
synced with merged pr 😏